code/game/jobs/job/command/auxiliary/cas_pilot.dm

```diff
@@ -0,0 +1,24 @@
+/datum/job/command/pilot/cas_pilot
+	title = JOB_CAS_PILOT
+	total_positions = 1
+	spawn_positions = 1
+	allow_additional = TRUE
+	scaled = TRUE
+	supervisors = "the auxiliary support officer"
+	flags_startup_parameters = ROLE_ADD_TO_DEFAULT
+	gear_preset = /datum/equipment_preset/uscm_ship/gp
+	entry_message_body = "<a href='"+WIKI_PLACEHOLDER+"'>Your job is to fly, protect, and maintain the ship's gunship.</a> While you are an officer, your authority is limited to the dropship, where you have authority over the enlisted personnel."
+
+/datum/job/command/pilot/whiskey
+	total_positions = 2
+	spawn_positions = 2
```

Comment:
This probably could have been done in code\game\gamemodes\colonialmarines\whiskey_outpost\equipping.dm instead but it works here.